### ISSUE TITLE
Pop from empty list error

### ISSUE THREAD

**Issue by sciencectn** [chaimleib/intervaltree#26]
Got another one for you. This one seemed to show up way faster than #25. 

```
t = intervaltree.IntervalTree()
t.addi(17.89,21.89)
t.addi(11.53,16.53)
t.removei(11.53,16.53)
t.removei(17.89,21.89)
t.addi(-0.62,4.38)
t.addi(9.24,14.24)
t.addi(4.0,9.0)
t.removei(-0.62,4.38)
t.removei(9.24,14.24)
t.removei(4.0,9.0)
t.addi(12.86,17.86)
t.addi(16.65,21.65)
t.removei(12.86,17.86)
```

I got this error:
```
    max_iv = ivs.pop()
IndexError: pop from empty list
```

**Comment by chaimleib** [comment]
Working on it now!

**Comment by chaimleib** [comment]
Is it fixed in 2.0.1 now?

**Comment by sciencectn** [comment]
Looks good, thanks!
